docs/remote-snapshotter.md

```diff
@@ -0,0 +1,127 @@
+# Remote Snapshotter
+
+Containerd allows snapshotters to reuse snapshots existing somewhere managed by them.
+
+_Remote Snapshotter_ is a snapshotter that leverages this functionality and reuses snapshots that are stored in a remotely shared place.
+These remotely shared snapshots are called _remote snapshots_.
+Remote snapshotter allows containerd to prepare these remote snapshots without pulling layers from registries, which hopefully shorten the time to take for image pull.
+
+One of the remote snapshotter implementations is [Stargz Snapshotter](https://github.com/containerd/stargz-snapshotter).
+This enables containerd to lazily pull images from standard-compliant registries leveraging remote snapshotter functionality and stargz images by google/crfs.
+
+## The containerd client API
+
+The containerd client's `Pull` API with unpacking-mode always attempts to query remote snapshots to the underlying snapshotter.
```

Comment:
`always attempts to query remote snapshots to the underlying snapshotter`

should probably be more like

`allows the underlying snapshotter to query for remote snapshots before fetching content`